azure-pipelines.yml

```diff
@@ -7,10 +8,24 @@ variables:
   value: '$(system.defaultWorkingDirectory)/gopath'
 - name: modulePath # Path to the module's code
   value: '$(GOPATH)/src/github.com/platform9/azure-provider'
+- name: KubernetesVersion
+  value: 'v1.10.0'
+- name: MinikubeVersion
+  value: 'v0.27.0'
 
 jobs:
 - job: BuildAndUnitTest
   pool:
     vmImage: 'Ubuntu 16.04'
   steps:
   - template: ./ci/azure-pipelines-build-test.yml
+
+# Runs end-to-end tests
+- job: E2ETest
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  dependsOn: BuildAndUnitTest
+  # TODO: remove before merge
```

Comment:
remove before merging. The E2E pipeline should only run on master merges for now.